arrays.js

```diff
@@ -4,62 +4,87 @@
 **************************************************************/
 function sumArray(numbers) {
   //TODO: Add your code here
+let result=numbers.reduce((sum,value)=>{return sum+value});
+return result;
 }
-// console.log(sumArray([4, 3, 2, 5, -10]));
 
+console.log("Task-1 :\nSummation Of [4, 3, 2, 5, -10] = "+sumArray([4, 3, 2, 5, -10]));
+console.log("**************************************************************");
+console.log("**************************************************************");
 /**************************************************************
  Task 2:
   Create a function findFirstStringStartingWithA that receives an array of strings as a parameter
   and returns the first string that starts with the letter given letter.
 **************************************************************/
 function findFirstStringStartingWithLetter(letter, strings) {
   //TODO: Add your code here
+  let result = strings.find( (s)=>  (s.toLowerCase()).startsWith(letter.toLowerCase())  ==true );
+  return result;
 }
-// console.log(
-//   findFirstStringStartingWithLetter("h", ["Memory", "Hello", "Happy"])
-// );
-
+console.log("Task-2 :\nThe First Item In [\"Memory\", \"Hello\", \"Happy\"] That Start With (h) : " +
+  findFirstStringStartingWithLetter("h", ["Memory", "hello", "Happy"])
+);
+console.log("**************************************************************");
+console.log("**************************************************************");
 /**************************************************************
  Task 3:
   Create a function isPresentIncluded that receives an array of presents as a parameter and uses the includes method to check 
   if a present is included in the array.
 **************************************************************/
 function isPresentIncluded(presentName, presents) {
   //TODO: Add your code here
+  console.log("Task-3 :\n");
+  if(presents.includes(presentName)==true)
```

Comment:
No need to use an if statement since the includes method already returns a boolean (true or false)